catalog/app/components/Pricing/index.js

```diff
@@ -54,70 +59,68 @@ const Styler = styled.div`
   }
 `;
 
+const Detail = styled.p`
+  text-align: right;
+`;
+
 const perUser = <span className="unit">per user / month</span>;
 
-function Pricing({ signUp, takeAction }) {
+function Pricing({ signUp, takeAction = true, title = 'Pricing' }) {
   return (
     <Styler>
-      <h1 id="pricing">Pricing</h1>
-      <Tabs>
-        <Tab label="Cloud">
-          <table>
-            <tbody>
-              <tr>
-                <th>Free</th>
-                <th>Individual</th>
-                <th>Teams</th>
-              </tr>
-              <tr className="price">
-                <td>
-                  <h2>$0 { perUser }</h2>
-                </td>
-                <td>
-                  <h2>$7 { perUser }</h2>
-                </td>
-                <td>
-                  <h2>$49* { perUser }</h2>
-                </td>
-              </tr>
-              <tr>
-                <td>
-                  Unlimited public packages<br />
-                </td>
-                <td>
-                  Unlimited public packages<br />
-                  Up to 1TB of private packages<br />
-                </td>
-                <td>
-                  Unlimited public packages<br />
-                  1TB+ of private packages<br />
-                  Dedicated registry and storage<br />
-                  Priority support<br />
-                  Team administration and auditing<br />
-                  * Sold in blocks of 10 users<br />
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </Tab>
-      </Tabs>
+      <h1 id="pricing">{title}</h1>
+      <table>
+        <tbody>
+          <tr>
+            <th>Free</th>
+            <th>Individual</th>
+            <th>Team</th>
+          </tr>
+          <tr className="price">
+            <td>
+              <h2>$0 { perUser }</h2>
+            </td>
+            <td>
+              <h2>$7 { perUser }</h2>
+            </td>
+            <td>
+              <h2>$49* { perUser }</h2>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              •&nbsp;Unlimited public packages<br />
```

Comment:
If only HTML had a tag to create lists...

Reply:
what makes this sad is it would take me 20+ min. to get `ol` and `li` looking this good. true story.